webxr/openxr/input.rs

```diff
@@ -116,6 +144,7 @@ pub struct OpenXRInput {
     action_buttons_left: Vec<Action<f32>>,
     action_buttons_right: Vec<Action<f32>>,
     action_axes_common: Vec<Action<f32>>,
+    supported_interaction_profiles: Vec<&'static str>,
```

Comment:
Rather than passing in profile names and performing a contains check every frame, should we just store a boolean instead?